contrib/checkbox-ce-oem/checkbox-provider-ce-oem/bin/gst_resources_generator.py

```diff
@@ -101,8 +99,11 @@ def _v4l2_video_decoder_md5_checksum_comparison_helper(
         gst_v4l2_video_decoder_md5_checksum_comparison scenario
         """
         name = "{}x{}-{}-{}".format(width, height, decoder_plugin, color_space)
-        golden_sample_file = "{}/video_golden_samples/{}.{}".format(
-            self._args.video_codec_testing_data_path, name, source_format
+        golden_sample_file = "{}/{}/{}.{}".format(
+            self._args.video_codec_testing_data_path,
+            self.VIDEO_GOLDEN_SAMPLES,
+            name,
+            source_format,
```

Comment:
```suggestion
        golden_sample_file = os.path.join(
            self._args.video_codec_testing_data_path,
            self.VIDEO_GOLDEN_SAMPLES,
            name,
            source_format,
```

Reply:
@stanley31huang Please review this commit.
https://github.com/canonical/checkbox/pull/1337/commits/eedee03c6a9859e7b0d506f4f57b8cf7ecf038f4